suffuse/node_fuse.go

```diff
@@ -3,22 +3,92 @@ package suffuse
 import (
   "github.com/suffuse/go-suffuse/suffuse/xattr"
   "os"
+  "time"
   "golang.org/x/net/context"
   "bazil.org/fuse/fs"
   f "bazil.org/fuse"
 )
 
+// Access checks whether the calling context has permission for
```

Comment:
I'll try to comment that way but let's try not to be too vigilant about matters which should in principle be handled by software. If it matters much, we should write software to do the comment formatting automatically, and if it doesn't it doesn't.